.github/ISSUE_TEMPLATE/server-crash---stacktrace.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Server crash / Stacktrace
 about: Report server crashes and/or scary stacktraces
+labels: "type: bug"
```

Comment:
I'm not sure if the bug label is appropriate for most server crashes, I believe it is better to not label them directly, since most of the issues with that template are not really bugs in the first place